src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int randomNumber = new Random().nextInt(100);
```

Comment:
same here!
move generation of random number to the method

but 100 is a magic number and it should be a constant field (final static)